### DISCUSSION TITLE
Getting java.net.UnknownHostException: Name or Service not known error while running load test.

### DISCUSSION THREAD

**Discussion by bhakarswayamavarapu** [Xceptance/XLT#473] [General]
Hi team,

Getting below error while running the load test for the commerce site in **preview** mode(production), but I can access the site running the scripts locally.
**java.net.UnknownHostException:  Name or service not known**

Screenshot:
![image](https://github.com/Xceptance/XLT/assets/140174174/79de6f85-6e6b-41e0-9bb2-76aed7599275)

Can you please suggest?
Thanks!

**Comment by rschwietzke** [comment]
Does the machine name resolve on the cloud machine you might have used? It tries to resolve an IPv6, you might not have an entry for that, so make sure your machine resolves IPv4 only. 

Also, you can tell XLT to ignore v6: https://docs.xceptance.com/xlt/release-notes/5_5_x/#discard-ipv6-addresses-received-from-dns-104

**Reply by bhakarswayamavarapu** [reply]
Hi @rschwietzke , 
Thanks for the quick response, 
For your information, we are able to perform load test on development and staging instances with the existing configurations. Only in production we getting the issue.

We are not able to find the configuration file to set the below, that was suggested by you 
![image](https://github.com/Xceptance/XLT/assets/140174174/7f04bfc6-0ac2-4d2f-b3c5-3367a4f6446c)

Could you help us to give the detailed information to set up the DNS configurations?

Thanks!

**Comment by rschwietzke** [comment]
You can use any configuration file. Typically, you find that setting in `default.properties` in your test suite. See our example: https://github.com/Xceptance/posters-advanced-loadtest-suite/blob/master/config/default.properties on line 121.
